### DISCUSSION TITLE
Telegram Harry Gorilla and Kozy Koala

### DISCUSSION THREAD

**Discussion by L-I-Am** [signalstickers/signalstickers#456] [Stickers requests]
Hi, I just moved from Telegram to signal and I really miss these sticker packages. I saw someone already ported the jellyfish from telegram so I guess it wouldn't be that hard to port some others? Thank you!
![image](https://user-images.githubusercontent.com/23050191/104033991-96cd0c80-51d0-11eb-82db-00742cff330d.png)
![image](https://user-images.githubusercontent.com/23050191/104034031-a3516500-51d0-11eb-9079-8e95942d7935.png)

**Comment by romainricard** [comment]
Hi! Are these stickers animated? If not, you can use the Telegram bot https://t.me/AdhesiveStickerBot to port stickers from TG to Signal!

**Reply by L-I-Am** [reply]
These stickers are indeed animated

**Reply by Klievan** [reply]
+1
